### DISCUSSION TITLE
Thanks for this gem

### DISCUSSION THREAD

**Discussion by pedrofurtado** [okuramasafumi/alba#318] [General]
@okuramasafumi Hi again! 😄 👋 

I just want to say thanks for your work on this gem. We was using the netflix/fast_json_api previously and we need to migrate to a new gem, and the only well-maintained gem that we found was yours. The other are abbandoned, or not stable.

🍻 🤝

**Comment by choosen** [comment]
you can migrate to community driven fork: https://github.com/jsonapi-serializer/jsonapi-serializer
